lib/HTTP/BrowserDetect.pm

```diff
@@ -141,65 +206,91 @@ our @MISC_TESTS = qw(
+    'SkypeUriPreview'                    => 'Skype URI Preview',
+    Swiftbot                             => 'Swiftbot',
+    W3C_Validator                        => 'W3C Validator',
+    WhatsApp                             => 'WhatsApp',
+    ahrefs                               => 'Ahrefs',
+    altavista                            => 'AltaVista',
+    apache                               => 'Apache http client',
+    askjeeves                            => 'AskJeeves',
+    baidu                                => 'Baidu Spider',
+    baiduspider                          => 'Baidu',
+    bingbot                              => 'Bingbot',
+    bitlybot                             => 'bitly',
+    curl                                 => 'curl',
+    'developers.google.com//web/snippet' => 'Google+ Snippet',
+    embedly                              => 'Embedly',
+    facebook                             => 'Facebook',
+    facebookexternalhit                  => 'Facebook',
+    flipboard                            => 'Flipboard',
+    getright                             => 'GetRight',
+    golib                                => 'Go language http library',
+    google                               => 'Googlebot',
+    googleadsbot                         => 'Google AdsBot',
+    googleadsense                        => 'Google AdSense',
+    googlebotimage                       => 'Googlebot Images',
+    googlebotnews                        => 'Googlebot News',
+    googlebotvideo                       => 'Googlebot Video',
+    googlefavicon                        => 'Google Favicon',
+    googlemobile                         => 'Googlebot Mobile',
+    indy                                 => 'Indy Library',
+    infoseek                             => 'InfoSeek',
+    ipsagent                             => 'Verisign ips-agent',
+    java                                 => 'Java',
+    lib                                  => 'lib',
+    linkchecker                          => 'LinkChecker',
+    linkedinbot                          => 'LinkedIn',
+    linkexchange                         => 'LinkExchange',
+    lwp                                  => 'LWP::UserAgent',
+    lycos                                => 'Lycos',
+    malware                              => 'Malware / hack attempt',
+    mj12bot                              => 'Majestic-12 DSearch',
+    msn                                  => 'MSN',
+    msnmobile                            => 'MSN Mobile',
+    msoffice                             => 'Microsoft Office',
+    nutch                                => 'Apache Nutch',
+    nuzzel                               => 'Nuzzel',
+    outbrain                             => 'Outbrain',
+    phplib                               => 'PHP http library',
+    'pinterest/0.'                       => 'Pinterest',
+    'pinterestbot'                       => 'Pinterest',
+    puf                                  => 'puf',
```

Comment:
What is this?

Reply:
It seems to be a really old bot.  It still exists in `robots.txt` files, but I wasn't able to get any more info from it.  I'll leave it in for now.  I don't see this being a problem with false positives at this point.